ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/mac-security.j2

```diff
@@ -23,11 +23,17 @@ mac security
       {{ key_cli }}
 {%             endif %}
 {%         endfor %}
+{%         if profile.mka.key_server_priority is arista.avd.defined %}
+      mka session key-server priority {{ profile.mka.key_server_priority }}
+{%         endif %}
 {%         if profile.mka.session.rekey_period is arista.avd.defined %}
       mka session rekey-period {{ profile.mka.session.rekey_period }}
 {%         endif %}
 {%         if profile.sci is arista.avd.defined(true) %}
       sci
 {%         endif %}
+{%         for l2_protocol in profile.l2_protocols | arista.avd.natural_sort %}
+      l2-protocol {{ l2_protocol }} {{ profile.l2_protocols[l2_protocol].mode }}
```

Comment:
this method will no longer work, since the key will have underscore instead of dash as CLI says. So please do the "if-tree" for every supported key. Same in docs.

Reply:
Updated to use if conditions for each known protocol